org.bitcoincore.bitcoin-qt.svg

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
It should be installed by the build system & hosted upsteram

Reply:
It can also be downloaded from the repo if you want to have that for now, just add it as a file source to the app module and install it on post-install.